plugin/matchit.vim

```diff
@@ -691,22 +691,19 @@ fun! s:MultiMatch(spflag, mode)
   " - TODO:  A lot of this is copied from s:Match_wrapper().
   " - maybe even more functionality should be split off
   " - into separate functions!
-  let open = substitute(s:pat, s:notslash . '\zs:.\{-}' . s:notslash . ',', '\\),\\(', 'g')
-  let open = '\(' . substitute(open, s:notslash . '\zs:.*$', '\\)', '')
+  let openlist = split(s:pat . ',', s:notslash . '\zs:.\{-}' . s:notslash . ',')
   let midclolist = split(',' . s:pat, s:notslash . '\zs,.\{-}' . s:notslash . ':')
   call map(midclolist, {-> split(v:val, s:notslash . ':')})
   let closelist = []
   let middlelist = []
   call map(midclolist, {i,v -> [extend(closelist, v[-1 : -1]),
         \ extend(middlelist, v[0 : -2])]})
-  let middle = join(middlelist, '\),\(')
-  if middle !=# ''
-    let middle = '\(' . middle . '\)'
-  endif
-  let close = join(closelist, '\),\(')
-  if close !=# ''
-    let close = '\(' . close . '\)'
-  endif
+  call map(openlist,   {i,v -> v =~# s:notslash . '\\|' ? '\(' . v . '\)' : v})
```

Comment:
perhaps use `\%(` here as well at the other similar positions?

Reply:
That might be better, even `\(` will be replaced to `\%(` later at https://github.com/chrisbra/matchit/pull/13/files#diff-13430418ad824a5eab20e9c92ce3a6c9R719 .

Reply:
Updated.

Reply:
thanks